blocks/core/ff_module/ff_module-responses/ff_module-responses.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+var React = require('react');
+
+module.exports = React.createClass({
+	propTypes: {
+	        recipientList: React.PropTypes.element.isRequired,
+	        responseDetails: React.PropTypes.element.isRequired
+	    },
+	render: function() {
+		return <div className='ff_module-responses'>
+			<div className='ff_module-responses__recipients'>
+				{this.props.recipientList}
+			</div>
+			<div className='ff_module-responses__content'>
+				{this.props.responseDetails}
+			</div>
+		</div>
+	}
+});
```

Comment:
They are basically the same! So I think what I should just is just move the .less styles(of the responses module) in the scrollable list to make it responsive. I don't think this ff_module-responses is needed anymore. I created it thinking it would do something different but it just added more complexity.